model/config.go

```diff
@@ -1632,7 +1641,7 @@ type MessageExportSettings struct {
 	BatchSize           *int
 
 	// formatter-specific settings - these are only expected to be non-nil if ExportFormat is set to the associated format
-	GlobalRelayEmailAddress *string
+	GlobalRelaySettings *GlobalRelayMessageExportSettings
```

Comment:
GlobalRelay wasn't released as a part of v4.7, so there shouldn't be any customers with this old config schema. Any developers that have the old schema will need to manually remove `GlobalRelayEmailAddress` from `config.json` to get the server to start.